src/diffusers/loaders/single_file.py

```diff
@@ -51,6 +51,7 @@ def build_sub_model_components(
     torch_dtype=None,
     **kwargs,
 ):
+    
```

Comment:
Unrelated change?

Reply:
somehow came after `ruff formatting` ...hmmmm did not intent to commit